pkgs/by-name/ca/cargonode/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  bzip2,
+  git,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargonode";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "xosnrdev";
+    repo = "cargonode";
+    rev = "refs/tags/${version}";
+    hash = "sha256-xzBLuQRyKmd9k0sbBFV5amtFWwKqXR0CEsRv8SHiJcQ=";
+  };
+
+  cargoHash = "sha256-v+Fs2VJrpnIOk9nPRanYYChlR7WOfkXF1kbYOKjOUYc=";
+
+  doCheck = false;
```

Comment:
If all tests are failed, add `doCheck = false`, otherwise partially disable them. Indicate reason (e.g. require network access) in both cases.

See https://nixos.org/manual/nixpkgs/unstable/#disabling-package-tests

Reply:
Done.